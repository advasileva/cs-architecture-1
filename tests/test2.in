10
-5 0 2 -4 -105 999 -1 0 11 100500