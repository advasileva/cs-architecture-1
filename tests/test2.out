0 0 2 1 -100 2 4 0 2 2